src/Controller/UI/MdAppController.php

```diff
@@ -46,19 +67,21 @@ public function defaultAction()
      *
      * This function called when Post occurs upon submitt of the MdApp form.
      *
-     * @param Request $request The Symfony request object.
-     * @param integer $id      The id of the Dataset to change the dataset status for.
+     * @param Request     $request     The Symfony request object.
+     * @param MdappLogger $mdappLogger The Pelagos Mdapp logger.
+     * @param integer     $id          The id of the Dataset to change the dataset status for.
      *
-     * @Route("/change-dataset-status/{id}")
-     * @Method("POST")
+     * @Route(
+     *      "/mdapp/change-dataset-status/{id}",
+     *      name="app_ui_mdapp_changedatasetstatus",
```

Comment:
Our old route names started with pelaogos_app (that was the app name), so any route using the old route name will not work. Also for being consistent, we should changed this route.

Reply:
changed back